week2/homework/src/exportToDos.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const { loadToDos, logScreen } = require('./Util');
+const path = require('path');
+const fs = require('fs').promises;
+
+const exportToDos = async filename => {
+  const file = path.join(__dirname, filename);
+  try {
+    const todos = await loadToDos();
+    const output = todos
+      .map(todo => [todo.index + '. ' + todo.body + ' ' + todo.complete].join(', '))
```

Comment:
Some of these are not needed, I think you may don't even use them and they will work just fine:
```suggestion
      .map(todo => todo.index + '. ' + todo.body + ' ' + todo.complete)
```

Reply:
I think,I forgot that while trying to accomplish.. Thanks